Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator+Joining.swift

```diff
@@ -128,7 +128,6 @@ extension WebRTCCoordinator.StateMachine.Stage {
 
                     if isFastReconnecting {
                         await coordinator.stateAdapter.publisher?.restartICE()
-                        await coordinator.stateAdapter.subscriber?.restartICE()
```

Comment:
why don't we need to restart ICE on the subscriber?

Reply:
Because of fast reconnection there are no changes, so t here is no need to do anything. That being said, if the peerConnection internally loses its connectivity (due to reasons) we are observing it and we are making a restartICE then (that operation happens [here](https://github.com/GetStream/stream-video-swift/blob/ab4e6c0771c2b0d6a9471debe26cbaa3a7e30421/Sources/StreamVideo/WebRTC/v2/StateMachine/Stages/WebRTCCoordinator%2BJoined.swift#L421))